app/components/project-long-description.js

```diff
@@ -1,13 +1,49 @@
 import Ember from 'ember';
 
+/**
+  `project-long-description` displays and allows editing of the long
+  long description for the project.
+
+  ## Default usage
+  ```handlebars
+    {{project-long-description project=project}}
+  ```
+
+  @module Components
+  @extends Ember.Component
+  @class project-long-description
+ */
+
 export default Ember.Component.extend({
   classNames: ['project-long-description'],
 
+  /**
+    A service that returns the users credentials.
+
+    @property credentials
+    @type Ember.Service
+   */
   credentials: Ember.inject.service(),
 
+  /**
+    @property inEditMode
```

Comment:
These properties are pretty simple but it would be great to still include a short description for the sake of the docs readability.